src/OrchardCore.Commerce.Web/OrchardCore.Commerce.Web.csproj

```diff
@@ -7,8 +7,8 @@
     <NodeJsExtensionsMarkdownAnalysisMode>solution</NodeJsExtensionsMarkdownAnalysisMode>
     <ExecDotnetPostcleanCommand>false</ExecDotnetPostcleanCommand>
     <ExecDotnetPrebuildCommand>false</ExecDotnetPrebuildCommand>
-    <NodeJsExtensionsGlobalESLintConfigurationDirectory></NodeJsExtensionsGlobalESLintConfigurationDirectory>
-    <NodeJsExtensionsCreateStylelintConfigurationFile></NodeJsExtensionsCreateStylelintConfigurationFile>
+    <NodeJsExtensionsGlobalESLintConfigurationDirectory>false</NodeJsExtensionsGlobalESLintConfigurationDirectory>
+    <NodeJsExtensionsCreateStylelintConfigurationFile>false</NodeJsExtensionsCreateStylelintConfigurationFile>
```

Comment:
This is also changed to false according to https://github.com/Lombiq/NodeJs-Extensions/blob/dev/Lombiq.NodeJs.Extensions/Docs/Styles.md?plain=1#L77